admin/test/installer/postgres/docker-compose-node1.yml

```diff
@@ -1,12 +1,22 @@
-postgres:
-  image: postgres
-  ports:
-     -  "5432:5432"
-  environment:
-     - "constraint:flocker-node==1"
-     - "POSTGRES_USER=flocker"
-     - "POSTGRES_PASSWORD=flocker"
-     - "PGDATA=/var/lib/postgresql/data"
-  volume_driver: flocker
-  volumes:
-     - 'postgres:/var/lib/postgresql/data'
+version: "2"
+
+volumes:
+  postgres:
+    driver: "flocker"
+    driver_opts:
+      size: "10GiB"
+      profile: "silver"
```

Comment:
@wallrj : switching to bronze might be slightly better since bronze is mentioned in the tutorial?